app/models/nomis/offender_base.rb

```diff
@@ -167,7 +167,11 @@ def load_case_information(record)
   private
 
     def handover
-      @handover ||= HandoverDateService.handover(self)
+      @handover ||= if pom_responsibility&.custody?
+                      HandoverDateService.handover(self)
+                    else
+                      HandoverDateService::NO_HANDOVER_DATE
+                    end
```

Comment:
Is there a reason why we don't make this `if/else` statement the responsibility of the `HandoverDateService`? Is the intention to avoid causing a dependency between the two services (i.e. the call to `pom_responsibility&.custody?`)? If so, that makes sense.

Reply:
Yes. It does feel like we have one service in 2 bits here - responsibility and handover are clesrly 2 sides of the same coin (but that wasn't clear for quite some time). It would probably make sense to put them closer together somehow (to reflect their cohesive nature)